src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/toolkit/resources/doclets_zh_CN.properties

```diff
@@ -240,11 +241,14 @@ doclet.Description=说明
 doclet.ConstantField=常量字段
 doclet.Value=值
 doclet.table_of_contents=目录
+doclet.Sort_lexicographically=按字典顺序对成员详细信息进行排序
+doclet.Sort_by_source_order=按源顺序对成员详细信息进行排序
 doclet.hide_sidebar=隐藏子工具栏
 doclet.show_sidebar=显示子工具栏
 doclet.filter_label=筛选内容（键入 .）
 doclet.filter_table_of_contents=筛选目录
 doclet.filter_reset=重置
+doclet.sort_table_of_contents=按字典顺序对成员详细信息进行排序
```

Comment:
Seems incorrectly copied from line 244.

Reply:
This was what I got back from the translation team. When looking at the English file's changes, the two values are "Sort member details lexicographically" and "Sort member details in lexicographical order". Even though the wording is slightly different, these phrases mean the same thing, so the Chinese translation being the same makes sense to me.

Reply:
Yes, you're right. I only noticed the key names are very different and thought the values should not be the same.